src/main/java/com/trashit/back/controllers/StatsController.java

```diff
@@ -0,0 +1,42 @@
+package com.trashit.back.controllers;
+
+import com.trashit.back.dto.garbage.GarbageCountGroupByDateDto;
+import com.trashit.back.dto.stats.UserRankingDto;
+import com.trashit.back.dto.user.UserCountGroupByDateDto;
+import com.trashit.back.services.StatsService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@Tag(name = "Stats controller")
+@RestController
+@RequestMapping("/stats")
+public class StatsController {
+
+    @Autowired
+    private StatsService statsService;
+
+    @GetMapping("/ranking")
+    @Operation(summary = "Le classement général de l'ensemble des utilisateurs")
+    public List<UserRankingDto> getGeneralRanking() {
+        return statsService.getGeneralRanking();
+    }
+
+    @GetMapping("/garbages")
+    @Operation(summary = "Statistique de l'ensemble des déchets jetés par les utilisateurs")
+    public List<GarbageCountGroupByDateDto> getGarbagesThrown() {
+        return statsService.getGarbageCountGroupByDate();
+    }
+
+    @GetMapping("/users")
+    @Operation(summary = "Statistique de le nombre d'inscris par jour")
```

Comment:
C'est le nombre d'inscrit cette stats là ? 
En tout cas y a un pb dans la phrase ^^
```suggestion
    @Operation(summary = "Afficher le nombre d'utilisateurs inscris")
```